packages/renderer/src/lib/markdown/micromark-listener-handler.ts

```diff
@@ -26,16 +26,33 @@ export function createListener(
     value?: unknown,
   ) => void,
 ) {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  return (e: any): void => {
+  return (e: unknown): void => {
+    let eventTarget: object | undefined = undefined;
```

Comment:
`object` ? what kind ?

Reply:
@axel7083 just a general object to reduce the number of repeated type checks as we assert that `e.target` is an object in line 33.

Reply:
```suggestion
    let eventTarget: EventTarget;
```

You are doing a if else bellow setting the value, so typescript is smart enough to let you type it properly, since you `return` if you can't set a value.